packages/flutter_tools/templates/app/.gitignore.tmpl

```diff
@@ -39,6 +39,3 @@ app.*.symbols
 
 # Obfuscation related
 app.*.map.json
-
-# Exceptions to above rules.
-!/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
```

Comment:
I'm a little confused. Will this path appear in a user's app?

Reply:
not at all

Reply:
this was a test in the flutter repo that we checked in a .packages file for, there was an exception in our gitignore

Reply:
Oh I'm sorry, it's been so long since I coded I forgot how to read a diff...

Reply:
:P